### PR TITLE
Update exporter mappings

### PR DESCRIPTION
**Pull request by trask**
Improvements to the exporter mappings.

Also a commit that fixes `CdsProfileFetcher` to use configured http proxy.

### CODE REVIEW

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java [hunk 1]
```diff
@@ -412,34 +402,41 @@ private static void setProperties(Map<String, String> properties, long timeEpoch
...
+            // if you use just HTTP, Breeze will remove appid from the target
+            telemetry.setType("Http (tracked component)");
+            telemetry.setTarget(target + " | " + targetAppId);
+        }
+
+        String url = removeAttributeString(attributes, SemanticAttributes.HTTP_URL);
         if (url != null) {
             try {
                 URI uriObject = new URI(url);
-                String target = createTarget(uriObject);
```
Comment (heyams) [inline agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:430]: createTarget appends port to be part of the target.. i don't see that get added back.. are we dropping port entirely?

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java [hunk 2]
```diff
@@ -553,14 +561,6 @@ private static boolean removeAttributeBoolean(Map<AttributeKey<?>, Object> attri
         }
     }
 
-    private static String createTarget(URI uriObject) {
```
Comment (heyams) [inline agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:556]: i was referring to this method.

**Review by heyams — APPROVED** [review]
👍

File: agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java (hunk 1 above)
Comment (trask) [reply agent/exporter/src/main/java/com/microsoft/applicationinsights/agent/Exporter.java:430]: good catch! yes, we want to keep the port, i will fix 👍

**Pull request merged**